test/test_respondent_controller.py

```diff
@@ -1215,6 +1215,30 @@ def test_put_email_verification_uses_case_insensitive_email_query(self):
             account_controller.put_email_verification(token)
             query.assert_called_once_with("test@example.test", db.session())
 
+    def test_email_verification_resets_password_reset(self):
+        self.populate_with_respondent()
```

Comment:
This returns a respondent, so you don't need the line below

Reply:
It looks like it might even do a token as well

Reply:
This has been updated